Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -197,6 +207,41 @@ function getTexts(node, path, unixName) {
 	});
 }
 
+function getEntireText(node,path){
+	$('#txtModal').on('show.bs.modal',function(event){
```

Comment:
this is still wrong. the event handler should be added once on page load, that's it.